src/library/Uno.Themes.WinUI.Markup/Theme/CalendarDatePicker.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using Windows.UI;
+using Microsoft.UI.Text;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup;
+using Uno.Extensions.Markup.Internals;
+
+using ResourceKeyDefinitionAttribute = Uno.Themes.WinUI.Markup.ResourceKeyDefinitionAttribute;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
```

Comment:
Do we need the `Theme` class?
Currently this is `Uno.Themes.Markup.Theme.CalendarDatePicker` which seems a bit redundant. Uno.Themes.Markup.CalendarDatePicker` is shorter